SwordPlugin.inc.php

```diff
@@ -286,5 +327,31 @@ public function getInstallSchemaFile() {
 	function getInstallEmailTemplatesFile() {
 		return ($this->getPluginPath() . '/emailTemplates.xml');
 	}
-}
 
+	/**
+	 * @see PKPPlugin::getInstallEmailTemplatesFile()
+	 */
+	function getInstallEmailTemplatesFile() {
+		return ($this->getPluginPath() . '/emailTemplates.xml');
+	}
+
+	/**
+	 * @see PKPPlugin::getInstallEmailTemplateDataFile()
+	 */
+	function getInstallEmailTemplateDataFile() {
```

Comment:
This function should be removed entirely, since the locale contents have been adapted to PO format.

Reply:
(Removed)